.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "development"
+    ],
+    "esbonio.sphinx.confDir": ""
```

Comment:
Is this to include Sphinx? What ever happened to what we were doing with that? 🤔